code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -563,25 +564,30 @@ func (sc *StorageSmartContract) commitMoveTokens(alloc *StorageAllocation,
 }
 
 func (sc *StorageSmartContract) commitBlobberConnection(
-	t *transaction.Transaction, input []byte, balances cstate.StateContextI) (
+	t *transaction.Transaction, input []byte, balances cstate.StateContextI, timings map[string]time.Duration) (
 	string, error) {
 
+	m := Timings{timings: timings, start: time.Now(), fnName: "commit_connection"}
```

Comment:
What are all the timings about?

Reply:
@Sriep this is for timing benchmarks

Reply:
@sum-it08 let's not add too many of those benchmark time logs, this could be added when for debugging or doing benchmark tests but should not go to production.